### ISSUE TITLE
[CHRGLOB] Bud Globals - Crash on Initilization

### ISSUE THREAD

**Issue by tuday2** [enusbaum/MBBSEmu#99]
**Module Information**
* Identifier: CHRGLOB
* Module Name: Bud Globals
* Version: 1.1

**Describe the bug**
Fails to initialize

**To Reproduce**
1. Add to modules.json
2. MBBSEMU fails to load module
3. Example of module where .DAT files **do not** autogenerate, had to create all 4 dat files manually from .VIR (CHRGLOB, CHRMACRO, CHRRCNT, CHRUGRAM)

**Expected behavior**
Module loads

**Screenshots**
```
Critical Exception has occurred:
System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter 'Unknown Exported Function Ordinal in MAJORBBS: 157')
   at MBBSEmu.HostProcess.ExportedModules.Majorbbs.Invoke(UInt16 ordinal, Boolean offsetsOnly)
   at MBBSEmu.HostProcess.ExecutionUnits.ExecutionUnit.ExternalFunctionDelegate(UInt16 ordinal, UInt16 functionOrdinal)
   at MBBSEmu.CPU.CpuCore.Tick()
   at MBBSEmu.HostProcess.ExecutionUnits.ExecutionUnit.Execute(IntPtr16 entryPoint, UInt16 channelNumber, Boolean simulateCallFar, Boolean bypassState, Queue`1 initialStackValues, UInt16 initialStackPointer)
   at MBBSEmu.Module.MbbsModule.Execute(IntPtr16 entryPoint, UInt16 channelNumber, Boolean simulateCallFar, Boolean bypassSetState, Queue`1 initialStackValues, UInt16 initialStackPointer)
   at MBBSEmu.HostProcess.MbbsHost.AddModule(MbbsModule module)
   at MBBSEmu.Program.Run(String[] args)
```

**Software Information:**
- OS: Windows Server 2019 Standard
- Telnet Client: Netrunner

**Comment by enusbaum** [comment]
Couple issues here:

1. There IS code in place that should handle the `.VIR` -> `.DAT` copying, but it's not firing apparently. We'll need to figure out what's going on there.

https://github.com/enusbaum/MBBSEmu/blob/master/MBBSEmu/Btrieve/BtrieveFileProcessor.cs#L104

2. Ordinal `MAJORBBS.157` is `DCDATE` which takes in a string date value and returns a packed/encoded `int` version of the date. The opposite process of `MAJORBBS.428` which is `NCDATE`

**Comment by tuday2** [comment]
> Couple issues here:
> 
> 1. There IS code in place that should handle the Btrieve `.VIR` -> `.DAT` copying, but it's not firing apparently. We'll need to figure out what's going on there.
> 
> https://github.com/enusbaum/MBBSEmu/blob/master/MBBSEmu/Btrieve/BtrieveFileProcessor.cs#L104

Ok, this part is fixed with PR #120
